client/network/test/src/sync.rs

```diff
@@ -1100,10 +1102,14 @@ fn syncs_state() {
 		assert!(!net.peer(1).client().has_state_at(&BlockId::Number(64)));
 
 		let just = (*b"FRNK", Vec::new());
-		net.peer(1)
+		let hashof60 = net
+			.peer(0)
```

Comment:
```suggestion
			.peer(1)
```
Nit. For sure in this particular test doesn't really matter, but just for the sake of formal correctness here I think you want to get the hash by querying `peer(1)`

Reply:
Blocks are pushed to `peer(0)` and then synced. The source of truth is then `peer(0)`.

Reply:
Yeah, here are equal and peer(1) "can query" peer(0) db directly. But we are using that hash over peer(1). In a "real world" scenario they are not guaranteed to be equal and the only source of truth for peer(1) is peer(1).

But nevermind. As I said in this test is not so relevant